Zend/zend_builtin_functions.c

```diff
@@ -59,9 +59,7 @@ zend_module_entry zend_builtin_module = { /* {{{ */
 
 zend_result zend_startup_builtin_functions(void) /* {{{ */
 {
-	zend_builtin_module.module_number = 0;
```

Comment:
This ``0`` may be lost. It wasn't used. Right?

Reply:
Yes, unless I change the implementation of zend_next_free_module to not add 1.